src/components/CourseTags/fetchCourseTagSuggestions.jsx

```diff
@@ -0,0 +1,7 @@
+const contains = (stringA, stringB) => stringA.toLowerCase().includes(stringB.toLowerCase());
```

Comment:
rename the file to utils. fetchSuggestion does not make sense here.

Reply:
Followed the pattern of other suggestions being fetched.